static/swagger/api_v3.json

```diff
@@ -62,16 +62,17 @@
       "in": "path",
       "description": "Page number of results to return, zero-indexed",
       "required": true,
-      "type": "number",
-      "format": "integer"
+      "type": "integer",
+      "minimum":0
     },
     "year": {
       "name": "year",
       "in": "path",
       "description": "Competition Year (or Season). Must be 4 digits.",
       "required": true,
-      "type": "number",
-      "format": "integer"
+      "type": "integer",
+      "minimum":1997,
+      "maximum":2020
```

Comment:
Not sure I like hard-coding the value of something like this that a) is not the current year and b) we might forget to update in the future.

If it's the current year, fine, we'll update it with everything else that needs to be updated year-to-year, but setting it to several years in the future puts me off.

Reply:
I can change the max to 9999 just to ensure that it will always be a 4 digit number